lib/tibber.ts

```diff
@@ -129,6 +132,8 @@ export class TibberApi {
     this.#homeySettings = homeySettings;
     this.#token = token;
     this.#homeId = homeId;
+    const attributes = getGlobalAttributes();
+    this.#userAgent = `Homey/${attributes.firmwareVersion} com.tibber/${attributes.appVersion}`;
```

Comment:
Super minor, but consider destructuring this:
```suggestion
    const { firmwareVersion, appVersion } = getGlobalAttributes();
    this.#userAgent = `Homey/${firmwareVersion} com.tibber/${appVersion}`;
```
for a little bit terser code

Reply:
That's miles better :)